core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -1049,6 +1050,12 @@ class HttpGoogleServicesDAO(
       .build()
   }
 
+  def getResourceBufferServiceAccountCredential: Credential = {
```

Comment:
Barring how long a built `GoogleCredential` is valid for, I'm wondering if we can do all of this just one time during `Boot`.  Tests might want to use different JSON files, but the running App will only ever get 1 credential file at startup, so we have no need to keep making this API call to Google, unless the credential expires, in which case we might be able to keep track of the age of the credential and request it only when necessary to avoid extra calls to Google.

Reply:
can change this to a `lazy val`

Reply:
changing to a lazy val